build.yaml

```diff
@@ -1138,7 +1124,7 @@ job_type:
                    *run_build_sh_script_on_vagrant_box,
                    *destroy_vagrant_box]
           }
-      clean_repo: false
+      dont_clean_repo: true
```

Comment:
Can you please add a comment explaining why we don't want to clean the repo here?